charts/csm-authorization/values.yaml

```diff
@@ -17,16 +17,16 @@ cert-manager:
 authorization:
   # images to use in installation
   images:
-    proxyService: dellemc/csm-authorization-proxy:v1.9.0
-    tenantService: dellemc/csm-authorization-tenant:v1.9.0
-    roleService: dellemc/csm-authorization-role:v1.9.0
-    storageService: dellemc/csm-authorization-storage:v1.9.0
+    proxyService: dellemc/csm-authorization-proxy:v1.10.0
+    tenantService: dellemc/csm-authorization-tenant:v1.10.0
+    roleService: dellemc/csm-authorization-role:v1.10.0
+    storageService: dellemc/csm-authorization-storage:v1.10.0
     opa: openpolicyagent/opa
     opaKubeMgmt: openpolicyagent/kube-mgmt:0.11
 
-  # base hostname for the ingress rules that expose the services
-  # the proxy-server ingress will use this hostname
-  # the role-service ingress will use role.hostname
+  # proxy-server ingress will use this hostname
+  # NOTE: additional hostnames can be configured in authorization.proxyServerIngress.hosts
+  # NOTE: proxy-server ingress is configured to accept IP address connections so hostnames are not required
```

Comment:
This note isn't accurate yet. The ingress needs a rule that doesn't specify a host:

https://github.com/dell/helm-charts/blob/905c3951da497421276b54540a092712dc66b2b1/charts/csm-authorization/templates/ingress.yaml#L52

which we don't have in this branch yet.

Reply:
Let me look into this, seems like this came from rebase. I did not make any changes in here. I will resolve it.